Meta/Lagom/Tools/CodeGenerators/LibUnicode/GenerateUnicodeData.cpp

```diff
@@ -616,6 +672,20 @@ struct SpecialCasing {
     Condition condition { Condition::None };
 };
 
+struct CodePointRange {
+    u32 first { 0 };
+    u32 last { 0 };
+};
+
+struct BlockName {
+    CodePointRange code_point_range {};
+    StringView display_name;
+};
+)~~~");
+    generator.set("block_display_names_size", String::number(unicode_data.block_display_names.size()));
+    generator.append(R"~~~(
+extern Array<BlockName, @block_display_names_size@> g_block_display_names;
```

Comment:
Does this need to be in the header? All other generated arrays are `static constexpr` in the .cpp file (and nit pick, they all begin with a `s_` prefix)